src/main/java/com/gettyimages/api/Credentials.java

```diff
@@ -74,12 +74,12 @@ public static Credentials GetInstance(String apiKey, String apiSecret, String us
     }
 
     public Token GetAccessToken() throws SdkException {
-        Calendar now = Calendar.getInstance();
-        now.add(Calendar.MINUTE, 5);
```

Comment:
Are you sure you want to remove line 78?

Reply:
I didn't.  It's still there.  It's just a little hard to see with all the inline comments.

Reply:
I don't like that I had to expand the entire file to see all the changes. 😠